module/actor/entity.js

```diff
@@ -69,4 +71,79 @@ export default class LexArcanaActor extends Actor
     {
 
     }
+
+    /* -------------------------------------------- */
+    /*                  Accessors                   */
+    /* -------------------------------------------- */
+    getSpecialty(peritiaId, speName)
+    {
+        const peritia = this.data.data.peritiae[peritiaId];
+        let idx = LexArcanaUtils.ObjectToArray(peritia.specialties).findIndex(item => item.name===speName);
+        return peritia.specialties[idx];
+    }
+
+    /* -------------------------------------------- */
+    /*              	Manipulators                */
+    /* -------------------------------------------- */
+    async addPeritiaSpecialty(peritiaId, name, modifier)
+    {
+        const currentSpecialties = LexArcanaUtils.ObjectToArray(duplicate(this.data.data.peritiae?.[peritiaId]?.specialties ?? []));
+        await super.update({[`data.peritiae.${peritiaId}.specialties`]: [...currentSpecialties, { "name":name, "defaultRoll": "", "modifier": modifier }] });
+    }
+    async removePeritiaSpecialty(peritiaId, key)
+    {
+        let currentSpecialties = duplicate(this.data.data.peritiae?.[peritiaId]?.specialties ?? []);
+        currentSpecialties = LexArcanaUtils.ObjectToArray(currentSpecialties).filter(item => item.name!==key);
```

Comment:
Because here we need the complete list, so we will add code somewhere else to save code here. For the moment its useless since we only need this here, but maybe later if we need to do the same thing we can come up with something